oap-server/exporter/pom.xml

```diff
@@ -35,6 +35,10 @@
             <artifactId>server-core</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>io.grpc</groupId>
+            <artifactId>grpc-testing</artifactId>
```

Comment:
I think you miss dependency scope. This is a test dependency.

Reply:
I found the scope was declared as test in the parent pom

Reply:
Yes, this should be fine